diff --git a/node_modules/assets-webpack-plugin/dist/lib/output/createOutputWriter.js b/node_modules/assets-webpack-plugin/dist/lib/output/createOutputWriter.js
index e5f4545..c7c5c88 100644
--- a/node_modules/assets-webpack-plugin/dist/lib/output/createOutputWriter.js
+++ b/node_modules/assets-webpack-plugin/dist/lib/output/createOutputWriter.js
@@ -85,7 +85,7 @@ module.exports = function (options) {
     }
 
     if (options.keepInMemory) {
-      localFs.mkdirp(options.path, mkdirCallback);
+      localFs.mkdir(options.path, { recursive: true }, mkdirCallback);
     } else {
       fs.mkdir(options.path, { recursive: true }, mkdirCallback);
     }
